docs/docs/contribute/new-command/build-command-logic.mdx

```diff
@@ -185,11 +185,6 @@ class SpoExampleListCommand extends Command {
 }
 ```
 
-this.optionSets.push({
-      options: ['userId', 'userName'],
-      runsWhen: (args) => args.options.userId || args.options.userName
-    });
-
 Option sets also allow you to define `runsWhen` for a set of options. This allows you to define a condition when the option set should be executed. E.g., when the option `title` is defined, either the option set `ownerGroupId` and `ownerGroupName` should be required.
 
 ```ts title="src/m365/planner/commands/plan/plan-set.ts"
```

Comment:
On line 199, I'm seeing a little enhancement.

Instead of using 

```ts
runsWhen: (args) => {
          return args.options.title !== undefined;
        }
```

we could write:

```ts
runsWhen: (args) => args.options.title !== undefined
```